content/docs/create-a-new-react-app.md

```diff
@@ -8,87 +8,87 @@ prev: add-react-to-a-website.html
 
 ```bash
 npx create-react-app my-app
 cd my-app
 npm start
 ```
 
->Note
+>Catatan
 >
->`npx` on the first line is not a typo -- it's a [package runner tool that comes with npm 5.2+](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b).
+>`npx` pada baris pertama bukanlah salah ketik --itu adalah [*package runner tool* yang tersedia bersamaan dengan *npm* 5.2+](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b).
 
-Create React App doesn't handle backend logic or databases; it just creates a frontend build pipeline, so you can use it with any backend you want. Under the hood, it uses [Babel](http://babeljs.io/) and [webpack](https://webpack.js.org/), but you don't need to know anything about them.
+*Create React App* tidak mengatur bagian *backend* ataupun basis data; namun hanya membuat sebuah *frontend build pipeline*. Sehingga, anda dapat menggunakan aplikasi anda dengan *backend* yang anda inginkan. *Create React App* menggunakan [*Babel*](http://babeljs.io/) dan [*Webpack*](https://webpack.js.org/), tapi anda tidak perlu tahu tentang semua itu.
 
-When you're ready to deploy to production, running `npm run build` will create an optimized build of your app in the `build` folder. You can learn more about Create React App [from its README](https://github.com/facebookincubator/create-react-app#create-react-app-) and the [User Guide](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#table-of-contents).
+Ketika anda siap untuk melanjutkan ke tahap produksi, menjalankan `npm run build` akan membuat *build* yang teroptimasi dari aplikasi anda dalam *folder* `build`. Anda dapat belajar lebih dalam mengenai *Create React App* dari [README](https://github.com/facebookincubator/create-react-app#create-react-app-) dan [Petunjuk Pengguna](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#table-of-contents) *Create React App*.
 
 ### Next.js {#nextjs}
 
-[Next.js](https://nextjs.org/) is a popular and lightweight framework for **static and server‑rendered applications** built with React. It includes **styling and routing solutions** out of the box, and assumes that you're using [Node.js](https://nodejs.org/) as the server environment.
+[*Next.js*](https://nextjs.org/) adalah *framework* popular dan ringan untuk aplikasi statis dan aplikasi *server-rendered* yang dibangun dengan *React*. Framework ini sudah mengatasi masalah ***styling*** dan ***routing***, dan mengasumsikan bahwa anda menggunakan [Node.js](https://nodejs.org/) sebagai *environment* server anda.
```

Comment:
*server-rendered* is fine to be translated as yang di-*render* di server I think, what do you think?